frontend/package.json

```diff
@@ -45,6 +45,7 @@
     "file-saver": "^2.0.5",
     "history": "^4.9.0",
     "immutable": "^3.8.2",
+    "jsoneditor": "^9.9.0",
```

Comment:
pls remove one of the two JSON editors, only one is used in your code. 
If possible keep `jsoneditor` (more mature), if you prefer to use a wrapper for the `jsoneditor` have a look at [here](https://www.npmjs.com/package/jsoneditor-react) this is a react wrapper for `jsoneditor`